sdk/core/azure-core/src/main/java/com/azure/core/implementation/ImplUtils.java

```diff
@@ -347,6 +350,17 @@ public static URL createUrl(String urlString) throws MalformedURLException {
         return new URL(urlString);
     }
 
+    @SuppressWarnings("unchecked")
+    public static <T> Class<? extends T> getClassByName(String className, Class<T> base) {
```

Comment:
Is `base` needed? It's not being used in this method.

Reply:
great catch!